book/02_getting_started.md

```diff
@@ -39,7 +40,7 @@ So, you've got Camping installed and it's running. Keep it running. You can
 edit files and The Camping Server will reload automatically. When you need to
 stop the server, press Control-C.
 
-Let's show something. At the bottom of nuts.rb add:
+Let's show something. At the bottom of camp.rb add:
 
 ```ruby
 module Nuts::Controllers
```

Comment:
No, in fact Camping now  looks for camp.rb by default, but it doesn't load the Camp Module.

When calling `Camping.goes :Nuts` Camping either fills the `Nuts` module with Camping's source, or creates a new module and fills that.

This does feel confusing because it breaks General ruby conventions. There is a larger issue that we need to figure out where to put controllers, helpers, models, etc... that break out from the single file paradigm. There is an issue that references that: https://github.com/camping/camping/issues/136.